### PR TITLE
Add Git mirror env var

### LINKED ISSUE #1229: BUILDKITE_REPO_MIRROR is not documented
Was added here, https://github.com/buildkite/agent/pull/1311.

Would expect it here, https://buildkite.com/docs/pipelines/environment-variables, but there doesn't seem to be a mention anywhere.

### PR DESCRIPTION
**Pull request by plaindocs**
Makes sense to link this up despite it being experimental.

Fixes  #1229

### CODE REVIEW

File: pages/pipelines/environment_variables.md.erb [hunk 1]
```diff
@@ -406,6 +406,13 @@ The repository of your pipeline. This variable can be set by exporting the envir
 
 Example: `"git@github.com:acme-inc/my-project.git"`
 
+<h3 class="h3-caps">BUILDKITE_REPO_MIRROR</h3>
+
+Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled.
```
Comment (karensawrey) [inline pages/pipelines/environment_variables.md.erb:411]: IMO it lacks the standard description element "This value can be set by doing X".

**Review by karensawrey — CHANGES_REQUESTED** [review]

File: pages/pipelines/environment_variables.md.erb (hunk 1 above)
Comment (plaindocs) [reply pages/pipelines/environment_variables.md.erb:411]:
```suggestion
Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled. The value cannot be modified.
```

File: pages/pipelines/environment_variables.md.erb [hunk 2]
```diff
@@ -406,6 +406,13 @@ The repository of your pipeline. This variable can be set by exporting the envir
 
 Example: `"git@github.com:acme-inc/my-project.git"`
 
+<h3 class="h3-caps">BUILDKITE_REPO_MIRROR</h3>
+
+Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled. The value cannot be modified.
```
Comment (karensawrey) [inline pages/pipelines/environment_variables.md.erb:411]:
```suggestion
Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled. The path to the shared git mirror. The value cannot be modified.
```

File: pages/pipelines/environment_variables.md.erb [hunk 3]
```diff
@@ -406,6 +406,13 @@ The repository of your pipeline. This variable can be set by exporting the envir
...
+The path to the shared git mirror.
```
Comment (karensawrey) [inline pages/pipelines/environment_variables.md.erb:412]:
```suggestion
```

**Review by karensawrey — CHANGES_REQUESTED** [review]
Tiny reorder for consistency is required.

**Review by karensawrey — APPROVED** [review]

File: pages/pipelines/environment_variables.md.erb [hunk 4]
```diff
@@ -406,6 +406,12 @@ The repository of your pipeline. This variable can be set by exporting the envir
 
 Example: `"git@github.com:acme-inc/my-project.git"`
 
+<h3 class="h3-caps">BUILDKITE_REPO_MIRROR</h3>
+
+Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled. The path to the shared git mirror. The value cannot be modified.
```
Comment (pda) [inline pages/pipelines/environment_variables.md.erb:411]: “The value cannot be modified” feels like it lacks context.
Maybe “… during a build” or “… after the agent has started” or something.
But I see this is an existing pattern, so maybe not important/relevant for this particular PR.

**Review by pda — APPROVED** [review]

File: pages/pipelines/environment_variables.md.erb (hunk 4 above)
Comment (plaindocs) [reply pages/pipelines/environment_variables.md.erb:411]: @pda yeah, that's mostly why I didn't mess with it. But I'd like to be more specific about what environment variables are writable when. 

What is the best way to get that info on a systematic basis?

**Pull request merged**